src/prefect/orion/api/deployments.py

```diff
@@ -320,7 +321,7 @@ async def create_flow_run_from_deployment(
                 status_code=status.HTTP_404_NOT_FOUND, detail="Deployment not found"
             )
 
-        parameters = deployment.parameters
+        parameters = get_default_values_from_parameter_schema_dict(deployment.parameter_openapi_schema)
```

Comment:
Maybe we should just have a `get_default_parameters()` method on the `Deployment` schema itself.